scripts/run-iqe-tests.sh

```
@@ -678,13 +688,17 @@ fi
+# update_exchange_rates to fetch rates. The get_daily_currency_rates Celery task
+# runs on a schedule and may not have executed yet on fresh/ephemeral deployments.
+#
+# Exec into the masu pod and curl localhost:8000 so the request never leaves the
+# pod: the operator's MasuNetworkPolicy only permits Prometheus on :9000, so any
+# off-pod caller to masu:8000 is denied. localhost traffic bypasses NetworkPolicy.
 echo ""
 echo "Seeding exchange rates via masu internal API..."
-MASU_INTERNAL_URL="http://${MASU_HOSTNAME}:${MASU_PORT}/api/cost-management/v1"
-EXCHANGE_RATE_RESPONSE=$(kubectl exec --request-timeout=30s -n "${NAMESPACE}" deploy/${HELM_RELEASE_NAME}-koku-api -c koku-api -- \
+MASU_INTERNAL_URL="http://localhost:${MASU_PORT}/api/cost-management/v1"
+EXCHANGE_RATE_RESPONSE=$(kubectl exec --request-timeout=30s -n "${NAMESPACE}" deploy/${HELM_RELEASE_NAME}-koku-masu -c masu -- \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)